external-import/cve/src/services/client/api.py

```diff
@@ -28,7 +30,7 @@ def _request_data(self, api_url: str, params=None):
         :return: Response in JSON format
         """
         try:
-            response = self.session.get(api_url, params=params)
+            response = self.retry_request(api_url, params)
```

Comment:
Can you not call it retry_request as it is a request with a retry mechanism?

Reply:
Yes, I renamed it :)